docs/account.mdx

```diff
@@ -0,0 +1,533 @@
+
+:::
+
+[address book entry]: ./addressbook.md
+
+## Network, ParaTime and Account selectors {#npa}
+
+Let's look at the three parameters you will often pass to any of the `account`
+commands.
+
+### Network
+
+The `--network <network_name>` parameter specifies the [network] which the
+Oasis CLI should connect to.
+
+For example:
+
+![code shell](../examples/account.04.in)
+
+![code](../examples/account.04.out)
+
+![code shell](../examples/account.05.in)
+
+![code](../examples/account.05.out)
+
+### ParaTime
+
+The `--paratime <paratime_name>` sets which [ParaTime] Oasis CLI should use.
+If you do not want to use any ParaTime, for example to perform a consensus
+layer operation, pass the `--no-paratime` flag explicitly.
+
+![code shell](../examples/account.06.in)
+
+![code](../examples/account.06.out)
+
+### Account
+
+The `--account <account_name>` specifies which account in your wallet the
+Oasis CLI should use to sign the transaction with.
+
+![code shell](../examples/account.07.y.in)
+
+![code](../examples/account.07.y.out)
+
+![code shell](../examples/account.08.y.in)
+
+![code](../examples/account.08.y.out)
+
+:::tip
+
+You can also set **the default [network][network-set-default],
+[ParaTime][paratime-set-default] or [account][wallet-set-default] to use**, if no
+Network, ParaTime or Account selectors provided respectively.
+
+:::
+
+[network]: ./network.md
+[paratime]: ./paratime.md
+[network-set-default]: ./network.md#set-default
+[paratime-set-default]: ./paratime.md#set-default
+[wallet-set-default]: ./wallet.md#set-default
+
+## Transfer {#transfer}
+
+Use `account transfer <amount> <to>` command to transfer funds between two
+accounts on the consensus layer or between two accounts inside the same
+ParaTime.
+
+![code shell](../examples/account.09.y.in)
+
+![code](../examples/account.09.y.out)
+
+![code shell](../examples/account.10.y.in)
+
+![code](../examples/account.10.y.out)
+
+## Allowance {#allow}
+
+`account allow <beneficiary> <amount>` command makes your funds withdrawable by
+a 3rd party beneficiary at consensus layer. For example, instead of paying your
+partner for a service directly, you can ask for their address and enable **them**
+to withdraw the amount which you agreed on from your account. This is a similar
+mechanism to how payment checks were used in the past.
+
+![code shell](../examples/account.11.y.in)
+
+![code](../examples/account.11.y.out)
+
+The allowance command uses relative amount. For example, if your run the
+above command 3 times, Logan will be allowed to withdraw 30 ROSE.
+
+:::tip
+
+To reduce the allowed amount or completely **disallow** the withdrawal, use the
+negative amount. To avoid flag ambiguity in the shell, you will first need to
+pass all desired flags and parameters except the negative amount, then append
+`--` to mark the end of options, and finally append the negative amount.
+
+```shell
+oasis account allow logan -- -10
```

Comment:
No example/test for this one?

Reply:
This one is tricky, because the example generator cannot append `--config` and `-y` flags to be non-interactive because of `--`.

Reply:
Ok.